tests/foreman/cli/test_discoveryrule.py

```diff
@@ -366,7 +356,7 @@ def test_positive_update_name(self):
         self.assertEqual(rule['name'], new_name)
 
     @run_only_on('sat')
-    @tier2
+    @tier1
```

Comment:
Same as in comment above, having functional type test on Acceptance level means it has to be moved to tier3.

Reply:
Updated Metadata as per your Doc.